plotly_resampler/figure_resampler/figure_resampler_interface.py

```diff
@@ -910,7 +912,7 @@ def construct_update_data(self, relayout_data: dict) -> List[dict]:
         return layout_traces_list
 
     def register_update_graph_callback(
```

Comment:
Shouldn't this be in the `FigureResampler` class instead of the abstract interface? IDK if this will ever be used by `FigureWidgetResampler`

Reply:
agree!